README.md

```diff
@@ -45,11 +55,11 @@ For you, a newcomer, to be running your development setup, you'll need to comple
 
 - Log in to <https://cloud.squidex.io/app/asap-hub-dev>
 - In `Content/Users`, create a new user (or edit if already created)
-- While creating, or editing the user, on `Functional/Connections`, add a new item to store your UUID (you can generate a random one [here](https://www.uuidgenerator.net/version4)) and save that (the save button is the blue one on top!). You'll need that UUID later, so save it.
+- While creating, or editing the user, on `Technical/Connections`, add a new item to store your UUID (you can generate a random one [here](https://www.uuidgenerator.net/version4)) and save that (the save button is the blue one on top!). You'll need that UUID later, so save it.
```

Comment:
This isn't true anymore. When you save a user a UUID is generated when you get the result back

Reply:
Thanks, I'll remove this